frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunTabDetails.tsx

```diff
@@ -15,32 +22,45 @@ import {
   DetailItem,
   renderDetailItems,
 } from '~/concepts/pipelines/content/pipelinesDetails/pipelineRun/utils';
-
 type PipelineRunTabDetailsProps = {
   pipelineRunKF?: PipelineRunKF;
   workflowName?: string;
+  loadError?: Error;
+  loaded: boolean;
 };
 
 const PipelineRunTabDetails: React.FC<PipelineRunTabDetailsProps> = ({
   pipelineRunKF,
   workflowName,
+  loadError,
+  loaded,
 }) => {
   const { namespace, project } = usePipelinesAPI();
 
-  if (!pipelineRunKF || !workflowName) {
+  if (!pipelineRunKF || !workflowName || !loaded) {
     return (
-      <EmptyState>
+      <EmptyState variant={EmptyStateVariant.large} data-id="loading-empty-state">
+        <Spinner size="xl" />
+        <Title headingLevel="h4" size="lg">
+          Loading
+        </Title>
+      </EmptyState>
+    );
+  }
+
+  if (loadError) {
+    return (
+      <EmptyState data-id="error-empty-state">
         <EmptyStateIcon icon={ExclamationCircleIcon} />
-        <Title headingLevel="h2" size="lg">
-          Error with the run
+        <Title headingLevel="h4" size="lg">
+          Error loading components
         </Title>
         <EmptyStateBody>
-          There was an issue trying to render the details information.
+          <EmptyStateBody>{loadError.message}</EmptyStateBody>
         </EmptyStateBody>
       </EmptyState>
```

Comment:
Couple things.

1. Do we need to really show the error on the drawer? Surely duplicating the error everywhere is not ideal (if anything, it should be shown on the center of the page) -- can we not get away with just understanding loading state of the page?
2. You doubled `EmptyStateBody` here

Reply:
@andrewballantyne updated the error handing in the drawer based on your comment. Can you check it again?